### PR TITLE
[GLUTEN-2047][DOC] Fix hdfs docs for build and configuration

### PR DESCRIPTION
**Pull request by Yohahaha**
## What changes were proposed in this pull request?

Update docs about hdfs support.

Fixes: #2047

### CODE REVIEW

**Comment by Yohahaha** [comment]
@zhouyuan could you help take a look on this?

**Review by zhouyuan — DISMISSED** [review]
👍

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -257,12 +200,6 @@ Here are two steps to enable kerberos.
 ```
 The ticket cache file can be found by `klist`.
 
-## 2.4 Yarn Cluster mode
```
Comment (zhouyuan) [inline docs/get-started/Velox.md:260]: Should keep this notes on Yarn mode?

File: docs/get-started/Velox.md [hunk 2]
```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
 
 ## 2.3 HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentatication like Kerberos based. Please note this library has serveral depedencies which may require extra installations on Driver and Worker node.
-On Ubuntu 20.04 the required depedencis are libiberty-dev, libxml2-dev, libkrb5-dev, libgsasl7-dev, libuuid1, uuid-dev. The packages can be installed via below command:
-```
-sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
```
Comment (PHILO-HE) [inline docs/get-started/Velox.md:159]: Not necessary dependency?

File: docs/get-started/Velox.md [hunk 3]
```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
...
-It is supported to access data on different HDFS endpoints.
-The endpoint info (hdfs://host:port) contained in a given hdfs file path will be used to initialize an hdfs client.
 
-If your HDFS is in HA mode, you need to set the [LIBHDFS3_CONF](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/depends/libhdfs3/src/client/Hdfs.cpp#L173) environment variable to specify hdfs client configuration file.
-An alternative, you can also make a [hdfs-client.xml](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/depends/libhdfs3/src/client/Hdfs.cpp#L185) and upload it to cluster.
+### Configuration about HDFS support
+ 
+HDFS uris (hdfs://host:port) will be extracted from a valid hdfs file path to initialize hdfs client, you do not need to specify it explicitly.
+
+libhdfs3 need a configuration file and has a little different from hdfs-site.xml and core-site.xml, you can find an example config file from [here](https://github.com/apache/hawq/blob/e9d43144f7e947e071bba48871af9da354d177d0/src/backend/utils/misc/etc/hdfs-client.xml).
```
Comment (PHILO-HE) [inline docs/get-started/Velox.md:170]: Suggestions:
needs a configuration file
it is a little different from

File: docs/get-started/Velox.md (hunk 2 above)
Comment (Yohahaha) [reply docs/get-started/Velox.md:159]: See this https://github.com/oap-project/gluten/blob/f7882f9d1478f9cc2a4381d69736a6ddb5372304/dev/package.sh#L18C54-L18C54

Comment (Yohahaha) [reply docs/get-started/Velox.md:159]: thirdparty-lib will include all needed dependencies.

File: docs/get-started/Velox.md (hunk 3 above)
Comment (Yohahaha) [reply docs/get-started/Velox.md]: Already updated.

File: docs/get-started/Velox.md [hunk 4]
```diff
@@ -153,31 +153,22 @@ mvn clean package -Pbackends-velox -Prss -Pspark-3.3 -DskipTests
 
 ## 2.3 HDFS support
 
-Hadoop hdfs support is ready via the [libhdfs3](https://github.com/apache/hawq/tree/master/depends/libhdfs3) library. The libhdfs3 provides native API for Hadoop I/O without the drawbacks of JNI. It also provides advanced authentatication like Kerberos based. Please note this library has serveral depedencies which may require extra installations on Driver and Worker node.
-On Ubuntu 20.04 the required depedencis are libiberty-dev, libxml2-dev, libkrb5-dev, libgsasl7-dev, libuuid1, uuid-dev. The packages can be installed via below command:
-```
-sudo apt install -y libiberty-dev libxml2-dev libkrb5-dev libgsasl7-dev libuuid1 uuid-dev
-```
-To build Gluten with HDFS support, below command is provided:
```
Comment (PHILO-HE) [inline docs/get-started/Velox.md:161]: Similarly, the build guide for s3 support is outdated. Could you please update that part also? See https://github.com/oap-project/gluten/blob/main/docs/get-started/Velox.md#25-aws-s3-support
Thanks!

Comment (zhouyuan) [reply docs/get-started/Velox.md:161]: @PHILO-HE I will follow-up on this

**Review by zhouyuan — APPROVED** [review]

**Pull request merged**